lib/utils/sanitize-html.js

```diff
@@ -0,0 +1,246 @@
+import validUrl from 'valid-url';
+import { filter } from 'lodash';
+
+/**
+ * Determine if a given tag is allowed
+ *
+ * @param {Node} node node being examined
+ * @returns {Boolean} whether the tag is allowed
+ */
+const isAllowedTag = node => {
+  const tagName = node.nodeName.toLowerCase();
+
+  if ('input' === tagName) {
+    return 'checkbox' === node.getAttribute('type');
```

Comment:
I noticed that checkboxes render with bullets, I think we should remove those so they look more like GitHub:

<img width="143" alt="screen shot 2018-03-15 at 3 19 05 pm" src="https://user-images.githubusercontent.com/789137/37494512-66138dee-2866-11e8-95cb-df0016c76cbf.png">

Reply:
I'd be more keen to fix this in a separate PR since this one right now is solely there for security. The rendering of bullets is going to be more related to how `showdown` generates the Markdown than how we sanitize it.

Reply:
ok 👍